src/ElmSharp/ElmSharp/Size.cs

```diff
@@ -53,10 +53,10 @@ public Size(int width, int height)
         /// </summary>
         /// <returns>The string is formatted as "{{Width={0} Height={1}}}".</returns>
         /// <since_tizen> preview </since_tizen>
-        public override string ToString()
-        {
-            return string.Format("{{Width={0} Height={1}}}", Width, Height);
```

Comment:
The behavior of 'string.Format(string, object, object)' could vary based on the current user's locale settings. Replace this call in 'Size.Test()' with a call to 'string.Format(IFormatProvider, string, params object[])'.